docs/apis-tools/community-clients/index.md

```diff
@@ -1,10 +1,29 @@
 ---
 id: index
-title: "Overview"
+title: "Community-supported component clients"
+sidebar_label: "Component clients"
 ---
 
-In addition to the core Java and Go clients provided by Zeebe, there are a number of community-maintained Zeebe client libraries:
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
 
+:::note
+Camunda extensions found in the [Camunda Community Hub](https://github.com/camunda-community-hub) are maintained by the community and are not part of the commercial Camunda product. Camunda does not support community extensions as part of its commercial services to enterprise customers.
+:::
+
+<Tabs groupId="clients" defaultValue="zeebe" queryString values={
+[
+{label: 'Zeebe', value: 'zeebe' },
+{label: 'Tasklist', value: 'tasklist' },
+{label: 'Operate', value: 'operate' },
+{label: 'Other', value: 'other' }
+]}>
+
+<TabItem value='zeebe'>
+
+In addition to the core Java and Go clients, there are a number of community-maintained Zeebe client libraries:
```

Comment:
Having text in one tab item but not the rest creates some visual bouncing. What do you think about moving this sentence above the tabs, and rewording to something more general? e.g. 

> In addition to the core Camunda-maintained clients, there are a number of community-maintained component libraries:

This is a non-blocking suggestion, but if it resonates and you want to make these changes, great.

Reply:
Great idea! I added it.